config/configmap/inferenceservice.yaml

```diff
@@ -50,6 +84,67 @@ data:
            "s3CABundle": ""
        }
     }
+  # ====================================== INGRESS CONFIGURATION ======================================
+  #  {
+       # ingressGateway specifies the ingress gateway to serve external traffic.
+       # The gateway should be specified in format <gateway namespace>/<gateway name>
+       # NOTE: This configuration only applicable for serverless deployment with Istio configured as network layer.
```

Comment:
I think the gateway is not Istio specific

Reply:
But it is only used with virtualservice creation. I am not sure.